src/argilla/client/feedback/dataset.py

```diff
@@ -605,6 +607,7 @@ def delete_dataset(dataset_id: UUID) -> None:
                 )
                 return
             self.argilla_id = argilla_id
```

Comment:
I guess here we could also return the current URL e.g.

```suggestion
            self.argilla_id = self.argilla_id or argilla_id
```